src/main/java/com/launchdarkly/client/LDConfig.java

```diff
@@ -22,6 +22,7 @@
   private static final int DEFAULT_SOCKET_TIMEOUT = 10000;
   private static final int DEFAULT_FLUSH_INTERVAL = 5;
   private static final long DEFAULT_POLLING_INTERVAL_MILLIS = 1000L;
+  private static final long DEFAULT_START_WAIT_MILLIS = 0L;
```

Comment:
I think the most sensible behavior is to set this to something like 5 seconds, as that's how we'd want most customers to use it.